bangumi_report.py

```diff
@@ -177,23 +177,67 @@ def __init__(self, image_url_list, user_id, year, type):
         if tmp_list:
             self.image_url_list.append(tmp_list)
 
-        self.env = Environment(loader=FileSystemLoader(os.getcwd()))
         self.user_id = user_id
         self.year = year
         self.type = type
 
-    def generate_report(self, to_stdout):
-        file_name = '%s-%s-%s-report.html' % (self.user_id, self.year, self.type)
-        logging.info('Output file: %s', file_name)
+    def generate_html(self, to_stdout):
+        from jinja2 import Environment, FileSystemLoader  # only import when generating HTML
+        self.env = Environment(loader=FileSystemLoader(os.getcwd()))
         logging.debug('Template file: %s', 'template.html')
         template = self.env.get_template('template.html')
-        html = template.render(user_id=self.user_id, image_list=self.image_url_list, year=self.year, type=self.type)
+        html = template.render(
+            user_id=self.user_id, image_list=self.image_url_list, year=self.year, type=self.type)
         if not to_stdout:
-            with open(file_name, 'w', encoding='utf-8') as f:
+            with open(self.file_name, 'w', encoding='utf-8') as f:
                 f.write(html)
         else:
             print(html)
 
+    def generate_markdown(self):
```

Comment:
Please make this function the same as `generate_html` to make it be able to print to stdout.

Reply:
Done.

Also, I modified the logging information when using `-o` flag.